enterprise/models.py

```diff
@@ -336,6 +335,13 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
         pending_link_record = PendingEnterpriseCustomerUser.objects.get(user_email=user_instance.email)
     except PendingEnterpriseCustomerUser.DoesNotExist:
         return  # nothing to do in this case
+    except (ProgrammingError, OperationalError):
+        logger.exception(
+            "DB-related error - skipping the rest of the handler. "
```

Comment:
@e-kolpakov -- can you summarize what the issue was that has been addressed by this PR?  I guess I'm not following why the `PendingEnterpriseCustomerUser` model/table would not be available.  Perhaps we could include a bit more info in this exception message, besides "Hey don't worry about this error, it's all good."